src/include/osl_pvt.h

```diff
@@ -629,6 +629,18 @@ class Symbol {
         m_lastwrite  = std::max (m_lastwrite, lw);
     }
 
+    // Mark the symbol as always being read (and, if write==true, also
+    // that it's always written). Then is for when we don't know when
+    // it's read or written, but want to be sure it doesn't look unused.
+    void mark_always_used (bool write=false) {
```

Comment:
For the current uses, it doesn't matter. It just needs to look used, doesn't matter if read or written. For generality and future use, I implemented the function to also have the ability to mark it as written, though I don't have a specific use case at this time.